packages/ndla-ui/src/Grid/Grid.tsx

```diff
@@ -6,102 +6,81 @@
  *
  */
 
-import { HTMLAttributes, ReactNode } from "react";
-import styled from "@emotion/styled";
-import { breakpoints, colors, misc, mq, spacing } from "@ndla/core";
+import { ReactNode } from "react";
+import { styled } from "@ndla/styled-system/jsx";
+
+const GridContainer = styled("div", {
+  base: {
+    display: "grid",
+    justifyContent: "center",
+    borderRadius: "xsmall",
+    padding: "xxlarge",
+    gridRowGap: "large",
+    gridColumnGap: "medium",
+    width: "100%",
+    marginBottom: "medium",
```

Comment:
Trenger vi margin? Jeg vil gjerne styre det fra artikkelen egentlig, se pr'en min

Reply:
Sikkert ikke om du styrer det fra et annet sted. Jeg bare hermet etter stylen som eksisterte jeg :^)

Reply:
Isåfall kan vi droppe det!